plugins/woocommerce/tests/php/includes/wc-core-functions-test.php

```diff
@@ -124,4 +124,12 @@ function() use ( $decimals ) {
 
 		remove_all_filters( 'wc_get_price_decimals' );
 	}
+
+	/**
+	 * Test test_wc_help_tip_strips_html() function.
+	 */
+	public function test_wc_help_tip_strips_html() {
+		$expected = '<span class="woocommerce-help-tip" tabindex="0" aria-label="Strong text regular text" data-tip="&lt;strong&gt;Strong text&lt;/strong&gt; regular text"></span>';
+		$this->assertEquals( $expected, wc_help_tip( '<strong>Strong text</strong> regular text' ) );
+	}
```

Comment:
It might be worthwhile to add a second test case here for when the second parameter of `wc_help_tip` is set to `true`.